website/docs/from_provider/quickstart.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Quickstart
+---
+
+## Try Riverpod online
+
+Before anything, we'd suggest you try out Riverpod in a sandbox to test its features out.
+
+The following example should be familiar and readable enough:
+  - dartpad.dev example (link)
+  - marvel app example, or another github repo example (link)
+  - ... more?
+
+## Migrating
+
+This section is designed for people familiar with the [Provider] package who
+wants to learn about Riverpod.
+
+Migrating from Provider to Riverpod can be very straightforward.  
+Migrating basically consists in a few steps that can be done in an *incremental* way.
+
+### Start gently
```

Comment:
As a title, it's not super clear I think.
I'd consider:

```suggestion
### Start with ChangeNotifierProvider
```


And the content likely would talk about how Riverpod voluntarily has ChangeNotifierProvider for easy migration.

Then we'd probably want to come back to this page to cross-link the StateNotifier->Notifier migration guide once it's out there. Let me update the draft for it

Reply:
We'd probably want a very quick note about autoDispose here too.
Something along the lines of "This applies to autoDispose. Although it is preferred to enable it, you can do that at a later stage".